### PR TITLE
CLI command to list stomp connections

### PR DESCRIPTION
**Pull request by dcorbacho**
Part of https://github.com/rabbitmq/rabbitmq-cli/issues/121

### CODE REVIEW

**Comment by dcorbacho** [comment]
The keys might need a second thought, are we exposing too much? too little?

**Comment by michaelklishin** [comment]
@dcorbacho it's comparable to MQTT. I think it's OK.

File: include/rabbit_stomp.hrl [hunk 1]
```diff
@@ -21,3 +21,31 @@
...
+         connection,
+         connection_state,
+         session_id,
+         channel,
+         version,
+         ssl_cert_login,
+         implicit_connect,
+         auth_login,
+         auth_mechanism,
+         ssl_login_name,
```
Comment (hairyhum) [inline include/rabbit_stomp.hrl]: `ssl_login_name` and `ssl_cert_login` should be covered by `auth_login` and `auth_mechanism`

**Comment by hairyhum** [comment]
I'm not sure about auth mechanisms naming. `default` is not descriptive enough. It should be `configuration`, or something that will make it clear that predefined configuration defaults are used.

**Pull request merged**